airflow/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -47,6 +48,8 @@ class RedshiftDataOperator(BaseOperator):
     :param with_event: indicates whether to send an event to EventBridge
     :param await_result: indicates whether to wait for a result, if True wait, if False don't wait
     :param poll_interval: how often in seconds to check the query status
+    :param return_sql_result: if True will return the result of an SQL statement,
+        if False will return statement ID
```

Comment:
Should we add that the default is False?

Reply:
+1

Reply:
Done